plugins/colorschemes/catppuccin.nix

```diff
@@ -199,6 +273,10 @@ in {
 
         noice = helpers.defaultNullOpts.mkBool config.plugins.noice.enable "";
 
+        # TODO: NormalNvim = ;
```

Comment:
How can I rename this option to snake_case?

Reply:
`normal_nvim` ?

Reply:
> `normal_nvim` ?

This integration's name is in camel case, will it work? https://github.com/catppuccin/nvim/commit/0e3c128eea8a7de692778d52b8429817df5c9040

Reply:
Oh, I didn't understand well. I would say that we leave it as `NormalNvim` (same as upstream).